common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -197,3 +198,40 @@ def assert_xss(self, response, xss_content):
         """
         self.assertContains(response, escape(xss_content))
         self.assertNotContains(response, xss_content)
+
+
+class MockSignalHandlerMixin(object):
```

Comment:
I'm also not a fan of the placement, but I couldn't figure out where a better place would be. Do you have any suggestions?